src/providers/dataProvider/dataprovider-utils.ts

```diff
@@ -66,6 +66,13 @@ export const auditForUpdatedChanges = async (
   subject?: User['id']
 ): Promise<UpdateParams<RCOResource>> => {
   const difference = getDifference(record.data, record.previousData)
+  const keys = Object.keys(difference)
+  if (keys.includes('dispatchedAt') && !difference?.dispatchedAt) {
```

Comment:
Hello @salony75way - to make the report more human-legible, is it possible to specify the value as `"unset"` instead of `null`?

Reply:
@IanMayo Should we only need to add "unset" to audit log or we need to set "unset" to db too? JFY we can't set "unset" to db as this field's data type is date. Please confirm this.

Reply:
oh, I thought we were just generating this field to go into the remarks text.

Reply:
Ok